code-scanning/properties/zscaler-iac-scan.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Zscaler IaC Scan",
+  "creator": "Zscaler CWP",
+  "description": "Scan your Infrastructure as Code files using Zscaler Infrastructure as Code (IaC) Scan app",
+  "iconName": "zscaler",
```

Comment:
Possible to have an icon only with logo and not text. The icon is shown in a small area and hence better not have any text in it.

Reply:
My bad

Reply:
@mvecha can also take this?